share/github-backup-utils/ghe-backup-config

```diff
@@ -66,6 +66,18 @@ for f in "$GHE_BACKUP_CONFIG" "$GHE_BACKUP_ROOT/backup.config" \
   fi
 done
 
+# Check that moreutils ts has been installed
+if ! echo "moreutils" | ts &>/dev/null; then
+  echo "Error: moreutils ts not found. Please install https://joeyh.name/code/moreutils" 1>&2
```

Comment:
```suggestion
  echo "Error: moreutils not found. Please install https://joeyh.name/code/moreutils" 1>&2
```

moreutils supplies `ts` so I don't think we don't need to call out `ts` specifically.

Reply:
Done